### PR TITLE
[jsk_topic_tools] Fix filename in doc: static_transform_republisher->static_tf_republisher 

### PR DESCRIPTION
**Pull request by nakane11**
File name static_transform_republisher.py in documentation is wrong, so usage written here does not work.

### CODE REVIEW

File: doc/jsk_topic_tools/scripts/static_tf_republisher.rst
Comment (iory) [inline doc/jsk_topic_tools/scripts/static_tf_republisher.rst:2]: Could you also change the `=` to ```==================```

**Review by iory — CHANGES_REQUESTED** [review]

**Review by knorth55 — APPROVED** [review]

**Comment by k-okada** [comment]
@knorth55 thank you for review and approve. When you approve, I got notification email, that is good chance to merge the PR, but if the CI not finished, I have to wait CI, but usually I forget about this PR. So if you approve AFTER CI passed, I can push merge button immediately.

**Comment by nakane11** [comment]
I found the same commit in #1709 eea60a5

**Review by iory — APPROVED** [review]

**Comment by iory** [comment]
> I found the same commit in #1709 eea60a5

This PR is good because the commit granularity focuses on modifying the document.

**Pull request merged**
